themes-default/slim/src/components/subtitle-search.vue

```diff
@@ -0,0 +1,258 @@
+                <button type="button" class="btn-medusa btn-success" @click="manualSearch">Manual</button>
+            </div>
+        </div>
+
+        <vue-good-table v-if="subtitles.length"
+            :columns="columns"
+            :rows="subtitles"
+            :search-options="{
+                enabled: false
+            }"
+            :sort-options="{
+                enabled: true,
+                initialSortBy: { field: 'score', type: 'desc' }
+            }"
+            styleClass="vgt-table condensed subtitle-table"
+            >
+            <template slot="table-column" slot-scope="props">
+                <span v-if="props.column.label == 'Download'">
+                    <span>{{props.column.label}}</span>
+                    <span class="btn-medusa btn-xs pull-right" @click="$destroy">hide</span>
+                </span>
+                <span v-else>
+                    {{props.column.label}}
+                </span>
+            </template>
+            <template slot="table-row" slot-scope="props">
+                <span v-if="props.column.field == 'provider'">
+                    <img :src="`images/subtitles/${props.row.provider}.png`" width="16" height="16"/>
+                    <span :title="props.row.provider">{{props.row.provider}}</span>
+                </span>
+                <span v-else-if="props.column.field == 'lang'">
+                    <img :title="props.row.lang" :src="`images/subtitles/flags/${props.row.lang}.png`" width="16" height="11"/>
+                </span>
+                <span v-else-if="props.column.field == 'filename'">
+                    <a :title="`Download ${props.row.hearing_impaired ? 'hearing impaired ' : ' '} subtitle: ${props.row.filename}`" @click="pickSubtitle(props.row.id)">
+                        <img v-if="props.row.hearing_impaired" src="images/hearing_impaired.png" width="16" height="16"/>
+                        <span class="subtitle-name">{{props.row.filename}}</span>
+                        <img v-if="props.row.sub_score >= props.row.min_score" src="images/save.png" width="16" height="16"/>
+                    </a>
+                </span>
+                <span v-else-if="props.column.field == 'download'">
+                    <a :title="`Download ${props.row.hearing_impaired ? 'hearing impaired ' : ' '} subtitle: ${props.row.filename}`" @click="pickSubtitle(props.row.id)">
+                        <img src="images/download.png" width="16" height="16"/>
+                    </a>
+                </span>
+                <span v-else>
+                    {{props.formattedRow[props.column.field]}}
+                </span>
+            </template>
+        </vue-good-table>
+    </td>
+</tr>
+</template>
+<script>
+
+import { mapState } from 'vuex';
+import { VueGoodTable } from 'vue-good-table';
+
+export default {
+    name: 'subtitle-search',
+    components: {
+        VueGoodTable
```

Comment:
It's a good idea to import all the components you use. In the future we might not have `StateSwitch` registered globally.
```suggestion
        StateSwitch,
        VueGoodTable
```